dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathCheckOption.java

```diff
@@ -123,21 +129,30 @@ private static Options configureOptions() {
     return options;
   }
 
-  static ImmutableList<Path> generateInputClasspath(CommandLine commandLine)
-      throws RepositoryException, ParseException {
-    setRepositories(commandLine);
-
+  static ImmutableList<Artifact> generateArtifacts(CommandLine commandLine)
```

Comment:
This method helps to prepare a list of entry point artifacts.

Reply:
You're not really _generating_ artifacts. You're reading the artifacts specified on the command line. What about calling this `parseArtifacts` or `getArtifacts` or something?

Reply:
do you need to push this change?

Reply:
Thanks. (Somehow it was reverted) Updated.